data/ideas.json

```diff
@@ -1,4 +1,10 @@
 [
+	{
```

Comment:
Hi @Shetteemah,
Please fix the indent and punt it to the last object of the array.